packages/worker/src/worker.ts

```diff
@@ -186,6 +198,40 @@ export class Worker {
     const nativeWorkerCtor: WorkerConstructor = this.nativeWorkerCtor;
     const compiledOptions = compileWorkerOptions(addDefaultWorkerOptions(options));
     const nativeWorker = await nativeWorkerCtor.create(compiledOptions);
+    return await this.bundleWorker(compiledOptions, nativeWorker);
+  }
+
+  /**
+   * Create a replay Worker.
+   */
+  public static async createReplay(options: ReplayWorkerOptions, history: History): Promise<void> {
+    const nativeWorkerCtor: WorkerConstructor = this.nativeWorkerCtor;
+    const fixedUpOptions = {
+      taskQueue: options.testName + '-' + this.replayWorkerCount,
+      workflowThreadPoolSize: 1,
```

Comment:
We should instead start the worker in debug mode which doesn't create a thread pool at all.
`debugMode` should be omitted from ReplayWorkerOptions.